assets/airports/lrop.json

```diff
@@ -0,0 +1,674 @@
+                ["TOSVI","UVALU","OP088","OP087","OP086","OP085","OP084","OP083", "OP082", "OP081", "DILAS"],
+                ["TOSVI","UVALU","OP088","OP087","OP086","OP085","OP084","OP083", "OP082", "OP081", "OBELA"]
+            ]
+        },
+        "SORDU1U" : {
+            "icao": "SORDU1U",
+            "name": "Sordu One Uniform",
+            "suffix": {"08R":"", "08L":""},
+            "entryPoints": {
+                "SOMOV": ["SOMOV" ],
+                "SORDU": ["SORDU"]
+            },
+            "body": [
+                 "TUTIX", 
+                ["TEVRO", "A70+|S240-"], 
+                ["OP083", "A35-|S220-"], 
+                "OP082", 
+                "OP081"
+            ], 
+            "rwy": {
+                "08R" : [["OBELA", "A25+|S210-"]],
+                "08L" : [["DILAS", "A25+|S210-"]]
+            },
+            "draw": [
+                ["SORDU","TUTIX","TEVRO","OP083", "OP082", "OP081", "DILAS"],
+                ["SORDU","TUTIX","TEVRO","OP083", "OP082", "OP081", "OBELA"]
+            ]
+        },
+        "OBUGA1U" : {
+            "icao": "OBUGA1U",
+            "name": "Obuga One Uniform",
+            "suffix": {"08R":"", "08L":""},
+            "entryPoints": {
+                "OBUGA": ["OBUGA"],
+                "KOMAN": ["KOMAN"]
+            },
+            "body": [
+                ["TEVRO", "A70+|S240-"], 
+                ["OP083", "A35-|S220-"], 
+                "OP082", 
+                "OP081"
+            ], 
+            "rwy": {
+                "08R" : [["OBELA", "A25+|S210-"]],
+                "08L" : [["DILAS", "A25+|S210-"]]
+            },
+            "draw": [
+                ["OBUGA","TEVRO","OP083", "OP082", "OP081", "DILAS"],
+                ["OBUGA","TEVRO","OP083", "OP082", "OP081", "OBELA"]
+            ]
+        }
+    },
+    
+    "spawnPatterns": [
+    	{
+    		"origin": "LROP",
+            "destination": "",
+            "category": "departure",
+            "route": "LROP.SOKRU1K.SOKRU",
+            "altitude": null,
+            "speed": null,
+            "method": "random",
+            "rate": 7,
+            "airlines": [
+               ["ROT", 5],
+               ["RYR", 4],
+               ["BAW", 1],
+               ["AFR", 1],
+               ["KLM", 1],
+               ["DLH", 1]
+            ]
+    	},
+    	{
+    		"origin": "LROP",
+            "destination": "",
+            "category": "departure",
+            "route": "LROP.BUKEL1K.BUKEL",
+            "altitude": null,
+            "speed": null,
+            "method": "random",
+            "rate": 7,
+            "airlines": [
+               ["ROT", 5],
+               ["RYR", 4],
+               ["BAW", 1],
+               ["AFR", 1],
+               ["KLM", 1],
+               ["DLH", 1]
+            ]
+    	},
+    	{
+    		"origin": "LROP",
+            "destination": "",
+            "category": "departure",
+            "route": "LROP.DENAK1K.KODRU",
```

Comment:
@georgeenciu The only restriction, as @indianbhaji pointed out, is that in a route of `ENTRY.PROCEDURE.EXIT`, the value of `ENTRY` and `EXIT` must be defined in the appropriate `entryPoints` or `exitPoints` section in that SID or STAR.

If you want the aircraft spawning on this pattern to "go to an exit point of the country", you can also do that by continuing the route, for example: `ENTRY.PROCEDURE.EXIT..FIXXA..FIXXB..FIXXC`.